entry/src/main/js/default/pages/index/index.js

```diff
@@ -32,10 +38,57 @@ export default {
         this.notifications = !this.notifications;
         console.log("Notifications toggled !");
     },
+    fetchDateAndTime : function(){
+        const date = new Date();
+        this.date_d=(String(date.getDate()).padStart(2, '0'))
+        this.date_m=(String(date.getMonth()+1).padStart(2, '0'))
+        this.time_h=(String(date.getHours()).padStart(2, '0'))
+        this.time_m=(String(date.getMinutes()).padStart(2, '0'))
+        this.time_h_0 = this.time_h.toString().substring(0, 1);
+        this.time_h_1 = this.time_h.toString().substring(1, 2);
+        this.time_m_0 = this.time_m.toString().substring(0, 1);
+        this.time_m_1 = this.time_m.toString().substring(1, 2);
+        const dayOfWeek = (date.getDay())
+        if (dayOfWeek==1)
+        this.date_w="Mon"
```

Comment:
Display Strings please maintain in json and ensure translation is handled for chinese. (converted string can be obtained from online translator)

Test UI both in english and chinese